go/cmd/dolt/commands/schcmds/tags.go

```diff
@@ -87,24 +89,58 @@ func (cmd TagsCmd) Exec(ctx context.Context, commandStr string, args []string, d
 			return 0
 		}
 	}
-	for i := 0; i < len(tables); i++ {
-		tables[i] = fmt.Sprintf("'%s'", tables[i])
+
+	var headerSchema = sql.Schema{
+		{Name: "table", Type: sql.Text, Default: nil},
+		{Name: "column", Type: sql.Text, Default: nil},
+		{Name: "tag", Type: sql.Uint64, Default: nil},
 	}
 
-	//TODO: implement REGEXP_SUBSTR in go-mysql-server and use it here instead of SUBSTR, as this will eventually break
-	queryStr := fmt.Sprintf("SELECT table_name AS 'table', column_name AS 'column', "+
-		"SUBSTR(extra, 5) AS tag FROM information_schema.columns WHERE table_name IN (%s)", strings.Join(tables, ","))
+	rows := make([]sql.Row, 0)
+
+	for _, tableName := range tables {
+		table, ok, err := root.GetTable(ctx, strings.ToLower(tableName))
+
+		if err != nil {
+			return 1
+		}
+
+		if ok {
```

Comment:
Should print an error and exit(1) if a given table cannot be found. (this logic will need to change when you make the above case sensitivity fix)

Reply:
I know the current semantics are to ignore any non-existent tables, but I think that's a mistake. It should fail loudly.